dumbo/src/tcp/connection.rs

```diff
@@ -1741,7 +1743,8 @@ pub(crate) mod tests {
 
         // Receiving data after the FIN is an error. We increase the rwnd edge for c, because the
         // window was full after the earlier reception tests.
-        c.advance_local_rwnd_edge(10_000);
+        let adv = c.advance_local_rwnd_edge(10_000);
```

Comment:
I think that adding a assert that also exercises the first branch of the `if` will make the coverage test pass.